R/site.R

```diff
@@ -12,6 +12,8 @@ render_site <- function(..., output_dir = "_site") {
   # absolute path of output_dir (which must exist)
   dir.create(output_dir, recursive = TRUE, showWarnings = FALSE)
   output_dir <- normalizePath(output_dir)
+  # clean up output dir if already present
+  unlink(output_dir, recursive = TRUE)
```

Comment:
I would not handle the existence of `output_dir` as this is passed to `render_book()`  which probably handles it better than we do, most likely checking if it exists an is not empty (which we should absolutely do if we unlink it at our end, since the user might pass a non-empty directory, maybe by mistake, and we should not just flush it, imagine using `output_dir = `~`)
```suggestion
```

Reply:
what `render_boook` does in case of a not-empty folder is to update the files the function creates, without removing anything else.  I had my `_site` folder full with the previous reports and those stayed there without impacting the creation of the website. As it is true that the user might use folders he/she does not want to flush, then it is probably safer to keep the behavior of `render_book`